samples/sampler/stories/default/ContextualPopupDecorator.js

```diff
@@ -0,0 +1,54 @@
+import {ContextualPopupDecorator} from '@enact/agate/ContextualPopupDecorator';
+import BodyText from '../../../../../moonstone/BodyText';
+import Button from '@enact/agate/Button';
+import ri from '@enact/ui/resolution';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import {boolean, select, text} from '@enact/storybook-utils/addons/knobs';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {action} from '@enact/storybook-utils/addons/actions';
+
+const ContextualButton = ContextualPopupDecorator(Button);
+ContextualButton.displayName = 'ContextualButton';
+
+const Config = mergeComponentMetadata('ContextualPopupDecorator', Button, ContextualButton);
+
+// NOTE: Something about the HOC is inhibiting accessing its defaultProps, so we're adding them here
+// manually. This can (should) be revisited later to find out why and a solution.
+Config.defaultProps = {
+	direction: 'below center',
+	open: false,
+	showCloseButton: true,
```

Comment:
Let's set `showCloseButton` of false or update the default value of the showCloseButton in the `ContextualPopupDecorator`. I hope that we could have the same default value between samplers and components.